libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<section>
+  <div id="map-section">
+    <tamu-gisc-drawer [width]="'25rem'" [position]="'left'">
+      <div class="content">
+        <h3>Map Controls</h3>
+        <label class="switch">
+          <input type="checkbox" id="csToggle" (click)="stateToggle(this)">
+          <span class="slider round"></span>
+        </label>
+        Counties/State
+        <br>
+        <label class="switch">
+          <input type="checkbox" id="mortalToggle" (click)="mortalityToggle(this)">
```

Comment:
Use `$event` instead of `this`. 

`this` refers to the component class instance.

`$event` refers to the event object.

Reply:
removing "this" and the parameter in the function declaration since it's not needed to perform the layer switch